src/rest/gmm_http_server.erl

```diff
@@ -36,7 +36,7 @@ start_server() ->
             %  POST {BODY->bulkRequest :: BulkEdgeCreationRequestDto} -> void
             {"/graph/edges/bulk", not_implemented, #{}},
 
-            %%% @todo rest_basic_queries - mostly implemented
+            %%% @todo rest_basic_queries
 
             %  POST {from, to} -> boolean
             {"/is_adjacent", rest_basic_queries, #{op => is_adjacent}},
```

Comment:
Czym jest `op`? Jeżeli to operacja, to:
```suggestion
            {"/is_adjacent", rest_basic_queries, #{operation => is_adjacent}},
```

Jest tego więcej, tamte też sprawdź

Reply:
Zmienione